tools/windows-node-installer/README.md

```diff
@@ -107,3 +109,29 @@ Global Flags:
 The `wni` destroys all resources (instances and security groups) specified in the `window-node-installer.json` file. 
 Security groups will not be deleted if they are still in-use by other instances.
 
+
+### Azure Platform 
+## Generate Service Principal:
+The wni tool requires a service principal credential file for Azure Platform to access the resources defined within the
+scope of the ID. To create one we do
+```bash
+az ad sp create-for-rbac --sdk-auth > azure.auth
+```
+This assumes that one have already set up the Azure CLI client, here we are creating one that is compatible with Azure SDK
+auth file to connect and authenticate with Azure.
+
+## Create Instance
+We can create instance with using azure create subcommand, Currently all the flags in create subcommand are optional.
+One can also provide already created resources such IP & NIC names via ipName, nicName towards node creation if you don't 
+want the installer to create one. The other arguments are image-id & instance-type which deals with setting up OS properties 
+and the size for the virtual machine. For more info on the details please do ./wni azure create --help
+
+# Sample command to create a window node:
+```bash
+./wni azure create --kubeconfig ~/OpenShift/azure/auth/kubeconfig --credentials /home/vinaykns/OpenShift/azure.auth --dir ~/windowsnodeinstaller/
```

Comment:
command execution result is pasted at  http://pastebin.test.redhat.com/802198

Reply:
Output still has `zapr@v0.1.0/zapr.go:69`. Please open a Jira story to fix if you are not able to figure this out quickly.

Reply:
Yup, there exists a jira story to fix this.